packages/@o3r/core/schematics/ng-add/schema.json

```diff
@@ -123,6 +123,11 @@
       "type": "boolean",
       "default": false,
       "alias": "g"
+    },
+    "withDevtool": {
+      "description": "Add option to automatically register the devtool module",
```

Comment:
a general question not necessarily linked to this PR: should we add links to the docs so that users can get more info during the script execution?

Reply:
I added the link for the preset because it required more details but not sure it is something that should be systematically done for all options.
Not sure it is needed for this one